corehq/apps/hqwebapp/static/cloudcare/scss/formplayer-webapp/form.scss

```diff
@@ -28,25 +28,18 @@
       margin-right: -@form-text-indent;
     }
 
-    background-color: lighten(@cc-brand-hi, 10);
+    background-color: @primary-subtle;
```

Comment:
@biyeun will `$primary-subtle: #DFE2H6` as mentioned in the [style guide](https://staging.commcarehq.org/styleguide/b5/atoms/colors/#interaction-attention) be available as a variable in b5? Brendon suggested using it but I resorted to `@call-to-action-extra-hi` since it was not currently defined.

Reply:
yeah, it'll be available, I think the way to do this in B5 is `background-color: var(--bsprimary-bg-subtle)`

Reply:
Thanks.

Reply:
I'm confused by this, aren't Sass variables referenced the same as in Less but with `$` instead of `@`? As far as I know `var()` is vanilla css syntax -- not that it wouldn't work, just that it seems excessive.